src/cromshell/utilities/cromshellconfig.py

```diff
@@ -138,23 +158,55 @@ def __load_cromshell_config_file(config_directory, config_file_name):
     return config_options
 
 
-def __get_cromwell_server():
+def __get_cromwell_server(config_options: dict):
     """Get Cromshell Server URL from configuration options"""
 
-    if not cromshell_config_options["cromwell_server"]:
+    if not config_options["cromwell_server"]:
         raise Exception(f'Cromshell config file is missing "cromwell_server"')
 
     LOGGER.info("Setting cromwell server to cromwell url from config file.")
-    LOGGER.info(cromshell_config_options["cromwell_server"])
+    LOGGER.info(config_options["cromwell_server"])
+
+    return config_options["cromwell_server"]
+
+
+def resolve_requests_connect_timeout(timeout_cli: int):
+    """Override the default request timeout duration.
 
-    return cromshell_config_options["cromwell_server"]
+    By default the timeout duration is 5 sec, however
+    if the option is specified in the cromshell config file or
+    command line then this overrides the default.
+    CLI > Config File > Default
```

Comment:
I think you priority here.
That actually got me thinking, should we take the longest out of the 3, or use this priority setting? I think both are valid though.